packages/client/verifiable-credentials/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@crossmint/verify-credentials",
-    "version": "1.1.2",
+    "version": "1.4.0",
```

Comment:
if this pr didnt have a breaking change (changing projectId + clientSecret => apiKey), it would be `1.1.3`

but, considering that i think a bump to `1.2.0` makes sense